packages/babel-plugin-polyfill-corejs3/test/fixtures/usage-global/array-at-proposals-corejs-3.8/output.mjs

```diff
@@ -1,3 +1,3 @@
-import "core-js/modules/esnext.array.at.js";
-import "core-js/modules/esnext.typed-array.at.js";
+import "core-js/modules/es.array.at.js";
+import "core-js/modules/es.typed-array.at.js";
```

Comment:
Same here, they are introduced in "3.17".

Reply:
Yep, compat data fixed in `3.17.1`.